src/Components/Footer/Footer.jsx

```diff
@@ -0,0 +1,30 @@
+import classes from './Footer.module.scss';
+import email from '../../assets/images/email.png';
+
+export const Footer = () => {
+  return (
+    <footer className={classes.footer}>
+      <div className={classes.mailContainer}>
+        <img className={classes.mail} src={email} alt="mail" />
+      </div>
+      <div className={classes.contactsContainer}>
+        <h2 className={classes.title}>Contacts:</h2>
+        <div className={classes.contacts}>
+          <a className={classes.contact}
+             href="https://github.com/MaksimkaSota"
+             target="_blank"
+             rel="noreferrer">
```

Comment:
It is also good idea to add "noopener" attribute =)
```
rel="noopener noreferrer"
```

Reply:
Done!